extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/filters/Filter.java

```diff
@@ -22,4 +22,11 @@ public interface Filter {
      */
     int getPriority();
 
+    /**
+     * @return true if filter should be applied on failures rather than HTTP requests
+     */
+    default boolean isFailureHandler() {
```

Comment:
Will that change any expectations, i.e, what happens right now to a filter, will it get called for the normal and error flows ?

Reply:
No, this flag is the semaphore - do I apply handler on HTTP requests (regular handler) or on failures (failure handler). I adjusted Javadoc, let see whether it's clearer now. Default behavior is backwards compatible.